tools/flaky-test-reporter/config.go

```diff
@@ -34,10 +34,28 @@ const (
 )
 
 var (
-	jobConfigs = []JobConfig{
-		{"ci-knative-serving-continuous", "serving", prow.PostsubmitJob}, // CI flow for serving repo
+	// jobConfigs lists all repos and jobs to run within those repos
+	jobConfigs = map[string][]JobConfig{
+		// CI flows for serving repo
+		"serving": {{Name: "ci-knative-serving-continuous", Type: prow.PostsubmitJob, PostIssue: true},
```

Comment:
Can we use the prowapi const here instead of defining our own? Looks like something existing. Feel free to no-op this.

Reply:
Not familiar with the prowapi. prow.PostsubmitJob references a const in shared/prow, is that what you mean?

Reply:
shared/prow is like our version of [prow](https://github.com/kubernetes/test-infra/tree/master/prow/apis/prowjobs/v1). I would like to use the stuff in that lib directly instead of recreating this in our version. Maybe not part of this PR, but in the future.

Reply:
Agreed. I will plan on looking at it down the road.